src/bin/TutorialD/tutd.hs

```diff
@@ -82,6 +82,9 @@ printWelcome = do
 
 main :: IO ()
 main = do
+  hSetEncoding stdin  utf8
```

Comment:
What issue does this fix? Is this a platform-specific fix? I have been able to print out UTF-8 characters without this.

Reply:
Thanks for the review.

Yes, it's a platform-specific fix. I use nixos in virtualbox and stack. And my stack don't parse locale settings. It turns en-US.utf8 into posix. If I don't add these lines, TutorialD command will fail. Haven't figure that out.

Reply:
I will take that off when I pull request. Sorry for the confusion.